src/openeo_test_suite/lib/process_runner/base.py

```diff
@@ -0,0 +1,39 @@
+from typing import List, Dict, Any
+
+class ProcessTestRunner:
+
+  # Returns a list of all processes supported by the backend
+  # Follows definition of the same function in the openEO Python client
+  def list_processes(self) -> List[Dict]:
+    pass
+  
+  # Returns single process description
+  # Follows definition of the same function in the openEO Python client
+  def describe_process(self, process_id: str) -> Dict:
```

Comment:
In Python it documentation is done with
```python
def foo(...):
    """
    docs here
    """
    ...
``` 
instead of comments before the `def`

Reply:
Fixed.